brigade/static/scss/core/_utilities.scss

```diff
@@ -10,6 +10,35 @@
 }
 
 // Text alignment
-.text-center {
-  text-align: center;
+.text-left { text-align: left; }
+.text-right { text-align: right; }
+.text-center { text-align: center; }
+.text-justify { text-align: justify; }
+
+@media (min-width: $mobile-up) {
+  .text-left-mobile-up { text-align: left; }
+  .text-right-mobile-up { text-align: right; }
+  .text-center-mobile-up { text-align: center; }
+  .text-justify-mobile-up { text-align: justify; }
+}
+
+@media (min-width: $tablet-up) {
+  .text-left-tablet-up { text-align: left; }
+  .text-right-tablet-up { text-align: right; }
+  .text-center-tablet-up { text-align: center; }
+  .text-justify-tablet-up { text-align: justify; }
+}
+
+@media (min-width: $midscreen-up) {
+  .text-left-midscreen-up { text-align: left; }
+  .text-right-midscreen-up { text-align: right; }
+  .text-center-midscreen-up { text-align: center; }
+  .text-justify-midscreen-up { text-align: justify; }
+}
+
+@media (min-width: $widescreen-up) {
+  .text-left-widescreen-up { text-align: left; }
+  .text-right-widescreen-up { text-align: right; }
+  .text-center-widescreen-up { text-align: center; }
+  .text-justify-widescreen-up { text-align: justify; }
```

Comment:
This creates a lot of waste. You only use one of these, and if you take this pattern to its extreme then there are tons of utility classes that are really hard to tell when they're possible to remove. I know we're not going for optimizing CSS performance, but also at the extreme it results in a lot of unnecessary rules and bandwidth. I'd propose making a class name for the brigade list, say `brigade-list__state` and then write a single rule:

```scss
.brigade-list__state {
  text-align: left;

  @media (min-width: $widescreen-up) {
    text-align: right;
  }
}
```